tools/inspect/apple_macro_check.cpp

```diff
@@ -90,8 +92,6 @@ namespace boost
                   line_start = it + 1; // could be end()
               }
           }
-
-          std::string lineloc = linelink(full_path, std::to_string(line_number));
```

Comment:
Why don't you use this overload for all of the checkers, then? What's so special about `apple_macro_check`?

Also, where is that `switch` you're referring to?